src/js/components/Marathon.jsx

```diff
@@ -44,21 +46,26 @@ var Marathon = React.createClass({
   },
 
   componentDidMount: function () {
-    this.onRouteChange();
+    PluginActions.requestPlugins();
 
+    this.onRouteChange();
     this.bindKeyboardShortcuts();
 
-    PluginStore.once(PluginEvents.BOOTSTRAP_COMPLETE, () => {
-      this.startPolling();
-    });
-
-    PluginStore.bootstrap();
+    PluginStore.on(PluginEvents.CHANGE, this.onPluginStoreChange);
   },
 
   componentWillReceiveProps: function () {
     this.onRouteChange();
   },
 
+  onPluginStoreChange: function () {
+    var pluginLoadingState = PluginStore.getPluginLoadingState();
```

Comment:
Just thought startPolling is trigger more then once accidentally. But I am wrong, am I?
This is controlled by the condition, right?
Could yoe maybe introduce an additional getter in the store, that is combining this, e.g. `isComplete` or so?
This is also good for line 121.